packages/mapper/lib/mapper.js

```diff
@@ -19,6 +19,7 @@ class Mapper {
     this.awaitLoc = []; // track locations where await is needed
     this.checkAwait = false;
     this.cursorAssigned = false;
+    this.databases = { test: new Database(this, 'test') };
```

Comment:
Do we have to memoize the database object?

Reply:
I'm not sure what specifically you mean by memoize. Previously we weren't saving db objects, but now we are so that when you switch back to a db the collection objects that get created through proxy usage stick around for autocomplete/etc.

Reply:
Yep i meant that (store it so we don't have to refetch later). Makes sense!